src/tools/siduck/siduck.cpp

```diff
@@ -62,7 +62,7 @@ ServerConnectionCallback(
         }
         break;
     case QUIC_CONNECTION_EVENT_SHUTDOWN_INITIATED_BY_PEER:
-        printf("[conn][%p] Shutdown by peer, 0x%llx\n", Connection, Event->SHUTDOWN_INITIATED_BY_PEER.ErrorCode);
+        printf("[conn][%p] Shutdown by peer, 0x%llx\n", Connection, (unsigned long long)Event->SHUTDOWN_INITIATED_BY_PEER.ErrorCode);
```

Comment:
I just merged a PR that deleted this whole file. Merge with master and just ignore all these changes please.